ops/charm.py

```diff
@@ -388,7 +388,8 @@ class RelationDepartedEvent(RelationEvent):
     This is the inverse of the :class:`RelationJoinedEvent`, representing when a
     unit is leaving the relation (the unit is being removed, the app is being
     removed, the relation is being removed). It is fired once for each unit that
-    is going away.
+    is going away for remaining units, and once for every remaining unit for the
```

Comment:
nitpick: "It is fired once for each unit that is going away for remaining units, and once for every remaining unit for the departing unit." is a bit troublesome to grok (maybe it is just me so feel free to disregard this suggestion). Would the following be accurate and acceptable "It is fired once for each unit that is going away. It is also fired for all the other units that remain".

Reply:
Tweaked it - should be more clear now.